aeacus-reports.pl

```diff
@@ -273,8 +274,9 @@ =head1 DESCRIPTION
     }
 }
 
+# Do not upload to SweStore on Irma
 my $swestoreFlag = "-swestore";
-if ($sisyphus->machineType() eq 'hiseqx'){
+if (hostname() eq 'irma'){
```

Comment:
Ah good one! My intention was to use regex matching not eq. I will fix it. 
You are supposed to be on vacation btw! ;)